eval/eval_tracker.py

```diff
@@ -314,7 +307,7 @@ def insert_eval_results(
                     session.add(eval_result)
                     eval_logger.info(f"Added {key}:{score} to the database.")
                 else:
-                    eval_logger.warning(f"Omitting '{key}' with score {score} (type: {type(score).__name__})")
+                    eval_logger.warning(f"Omitting '{key}' with (type: {type(score).__name__})")
             session.commit()
         except Exception as e:
             session.rollback()
```

Comment:
Thanks @RyanMarten! you probably also need to update the update_evalresults_db function to look up for the model name in DB and update the existing model instance if it already exists in the DB